.circleci/config.yml

```diff
@@ -269,15 +255,6 @@ workflows:
             - build-and-push-live
           <<: *filter-main
 
-      - tariff/smoketests:
-          name: smoketest_staging
-          context: trade-tariff-testing
-          url: https://staging.trade-tariff.service.gov.uk
-          yarn_run: staging-tariff-admin-smoketests
-          requires:
-            - apply-terraform-staging
-          <<: *filter-main
```

Comment:
Why are we removing staging smoketests? 👀

Reply:
I was being a bit handwavy here and just assuming they needed looking at

Reply:
Yeah, they do. staging admin is also broken in places for me